vizro-core/tests/unit/vizro/models/_components/form/test_slider.py

```diff
@@ -0,0 +1,201 @@
+"""Unit tests for hyphen.models.slider."""
+import json
+
+import plotly
+import pytest
+from dash import dcc
+from pydantic import ValidationError
+
+import vizro.models as vm
+
+
+@pytest.fixture()
+def expected_slider():
+    return dcc.Slider(
+        min=0,
+        max=10,
+        step=1,
+        marks={},
+        value=5,
+        included=False,
+        className="slider_control",
+        id="slider_id",
+        persistence=True,
+    )
+
+
+class TestSliderInstantiation:
+    """Tests model instantiation."""
+
+    def test_create_slider_mandatory(self):
+        slider = vm.Slider()
+
+        assert hasattr(slider, "id")
+        assert slider.type == "slider"
+        assert slider.min is None
+        assert slider.max is None
+        assert slider.step is None
+        assert slider.marks is None
+        assert slider.value is None
+        assert slider.title is None
+        assert slider.actions == []
+
+    @pytest.mark.parametrize(
+        "min, max",
+        [
+            (0, None),
+            (None, 10),
+            (0, 10),
+        ],
+    )
+    def test_valid_min_max(self, min, max):
+        slider = vm.Slider(min=min, max=max)
+
+        assert slider.min == min
+        assert slider.max == max
+
+    def test_invalid_min_max(self):
+        with pytest.raises(
+            ValidationError,
+            match="Maximum value of slider is required to be larger than minimum value.",
+        ):
+            vm.Slider(min=10, max=0)
+
+    @pytest.mark.parametrize(
+        "value",
+        [
+            5,
+            6.5,
+            0,
+            10,
+        ],
+    )
+    def test_valid_value(self, value):
+        slider = vm.Slider(min=0, max=10, value=value)
+
+        assert slider.value == value
+
+    def test_invalid_value_low(self):
+        with pytest.raises(
+            ValidationError,
+            match="Please provide a valid value larger than the minimum value.",
+        ):
+            vm.Slider(min=1, max=10, value=0)
+
+    def test_invalid_value_high(self):
+        with pytest.raises(
+            ValidationError,
+            match="Please provide a valid value smaller than the maximum value.",
+        ):
+            vm.Slider(min=1, max=10, value=11)
```

Comment:
Would combine the error message here as proposed before and then parameterise the input going beyond the range of min/max?